Add unbounded media timed events to gap analysis and recommendations

chrisn opened the pull request:
See #39


<!--
    This comment and the below content is programatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/me-media-timed-events/pull/46.html" title="Last updated on Apr 1, 2020, 2:24 PM UTC (749274e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/me-media-timed-events/46/5ebcc6e...749274e.html" title="Last updated on Apr 1, 2020, 2:24 PM UTC (749274e)">Diff</a>

rjksmith commented:
Apologies for the delay. The short answer is: ok.

The longer answer is: I think adding the [use cases described by @mavgit](https://github.com/w3c/me-media-timed-events/issues/39#issuecomment-494596204) would be helpful to add context for the 'updating event times' recommendation. Also, I think they may be already supported by the API, but we should capture this and investigate.

Perhaps 'updating media timed events' would be a clearer heading.

chrisn commented:
I have changed the recommendations to allow for updating media timed events, and added a recommendation for the user agent to notify the web application of any changes to an event (timeline position or data payload).
